sympy/core/relational.py

```diff
@@ -1075,9 +1081,383 @@ def _n2(a, b):
             return dif
 
 
+# helpers for _eval_is_ge below
+def _invpow(p):
+    # invert a power and avoid evaluation
+    if p.is_Pow:
+        if p.exp == -1:
+            return p.base
+        return Pow(p.base, -p.exp, evaluate=False)
+    else:
+        return 1/p
+        # or should this be done?
+        from sympy import fraction
+        n, d = fraction(p)
+        if d is S.One:
+            return Pow(n, -1, evaluate=False)
+        if n.is_Number:
+            return d/n
+        return Mul(d, Pow(n, -1, evaluate=False), evaluate=False)
+
+# 0 <= x <= 1
+_small_pos = lambda x: fuzzy_and((
+    is_ge(x, S.Zero), is_ge(S.One, x)))
+
+# -1 <= x <= 0
+_small_neg = lambda x: fuzzy_and((
+    is_ge(x, S.NegativeOne), is_ge(S.Zero, x)))
+
+# b**e >= 1
+_big_ppow = lambda b, e: fuzzy_or((
+        fuzzy_and((b.is_zero, e.is_zero)),  # 1
+        fuzzy_and((Mod(e, 2) == 0, fuzzy_or((
+        is_ge(S.NegativeOne, b) and is_ge(e, S.Zero),
+        _small_neg(b) and is_ge(S.Zero, e))))),
+    _small_pos(b) and is_ge(S.Zero, e),
+    is_ge(b, S.One) and is_ge(e, S.Zero)))
```

Comment:
I still think these should be proper functions with docstrings and tests.

Reply:
They don't need to be "public" but the meaning should be clearer in the code.

Reply:
Yes -- once this is pared down as much as possible I will add final touches.